recipes/cunit/all/conanfile.py

```diff
@@ -110,14 +121,14 @@ def build(self):
         with self._build_context():
             with tools.chdir(self._source_subfolder):
                 self.run("autoreconf -fiv".format(os.environ["AUTORECONF"]), win_bash=tools.os_info.is_windows)
```

Comment:
This line has been driving me nuts... please 🙏 

```suggestion
                self.run("{} -fiv".format(tools.get_env("AUTORECONF")), win_bash=tools.os_info.is_windows)
```